src/sys/socket/consts.rs

```diff
@@ -1,75 +1,159 @@
 pub use self::os::*;
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
 mod os {
     use libc::{self, c_int, uint8_t};
 
     pub const AF_UNIX: c_int  = libc::AF_UNIX;
     pub const AF_LOCAL: c_int = libc::AF_LOCAL;
     pub const AF_INET: c_int  = libc::AF_INET;
     pub const AF_INET6: c_int = libc::AF_INET6;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const AF_NETLINK: c_int = libc::AF_NETLINK;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const AF_PACKET: c_int = libc::AF_PACKET;
+    #[cfg(any(target_os = "macos", target_os = "ios"))]
+    pub const AF_SYSTEM: c_int = libc::AF_SYSTEM;
+    #[cfg(any(target_os = "macos", target_os = "ios"))]
+    pub const AF_SYS_CONTROL: c_int = 2;
 
     pub const SOCK_STREAM: c_int = libc::SOCK_STREAM;
     pub const SOCK_DGRAM: c_int = libc::SOCK_DGRAM;
     pub const SOCK_SEQPACKET: c_int = libc::SOCK_SEQPACKET;
     pub const SOCK_RAW: c_int = libc::SOCK_RAW;
+    #[cfg(not(any(target_os = "linux", target_os = "android")))]
+    pub const SOCK_RDM: c_int = libc::SOCK_RDM;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const SOCK_RDM: c_int = 4;
 
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const SOL_IP: c_int = libc::SOL_IP;
     pub const SOL_SOCKET: c_int = libc::SOL_SOCKET;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const SOL_TCP: c_int = libc::SOL_TCP;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const SOL_UDP: c_int = 17;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const SOL_IPV6: c_int = libc::SOL_IPV6;
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     pub const SOL_NETLINK: c_int = libc::SOL_NETLINK;
     pub const IPPROTO_IP: c_int = libc::IPPROTO_IP;
     pub const IPPROTO_IPV6: c_int = libc::IPPROTO_IPV6;
     pub const IPPROTO_TCP: c_int = libc::IPPROTO_TCP;
+    #[cfg(any(target_os = "linux", target_os = "android", target_os = "dragonfly"))]
     pub const IPPROTO_UDP: c_int = SOL_UDP;
+    #[cfg(any(target_os = "macos", target_os = "freebsd", target_os = "ios", target_os = "openbsd", target_os = "netbsd"))]
```

Comment:
Should I split `target_os`  in several lines for each case, or only when the line is longer than 100 characters ?